src/tokens-dist/colors.ts

```diff
@@ -150,7 +150,7 @@ export const EdsThemeColorButtonTertiaryBackgroundActive = '#21272D';
 export const EdsThemeColorButtonTertiaryBorder = '#707070';
 export const EdsThemeColorButtonTertiaryBorderHover = '#707070';
 export const EdsThemeColorButtonTertiaryBorderActive = '#21272D';
-export const EdsThemeColorButtonTertiaryText = '#6B65E2';
+export const EdsThemeColorButtonTertiaryText = '#707070';
```

Comment:
I don't know why this is showing up in this PR.. All the other tertiary text tokens are correct ('#707070'), but for some reason this one is still purple.

Reply:
Also, I didn't manually make this change, or the other change in this file, but the updates are correct.